src/main/webapp/app/entities/programming-exercise/test-cases/programming-exercise-manage-test-cases.scss

```diff
@@ -1,6 +1,3 @@
-@import '../../../../content/scss/_bootstrap-variables.scss';
```

Comment:
I wondered if the importing of the bootstrap css was a huge issue, but it didn't seem to be.